src/assets/scripts/client/parsers/aircraftCommandParser/AircraftCommandParser.js

```diff
@@ -299,7 +275,11 @@ export default class AircraftCommandParser {
      * @private
      */
     _validateCommandArguments() {
-        return _compact(_map(this.commandList, (command) => {
+        const validatedCommandList = _map(this.commandList, (command) => {
+            if (typeof command === 'undefined') {
+                return;
```

Comment:
this should still return a falsey value like `null` because using the `compact` below will remove those values properly.  just makes things more clear.  yay verbosity!

Reply:
done